frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -625,7 +645,10 @@ export const transformTopologyData = (
         if (!_.some(topologyGraphAndNodeData.graph.nodes, { id: uid })) {
           const operatorBacked = dataToShowOnNodes[uid].operatorBackedService;
           const nodeType = operatorBacked ? TYPE_OPERATOR_WORKLOAD : TYPE_WORKLOAD;
```

Comment:
You can assign the `HELM_WORKLOAD_TYPE` here. No need to pass the helm Resource map to `getTopologyNodeItem()`.
Something like this would work:
```suggestion
          const nodeType = operatorBacked ? TYPE_OPERATOR_WORKLOAD : isHelmReleaseNode? HELM_WORKLOAD_TYPE :TYPE_WORKLOAD;
```

Reply:
done.